hlxsites/merative2
# fix(blog-landing):fix design QA bugs to ensure the page aligns with the Design specs

## Issue

Fixes https://github.com/hlxsites/merative2/issues/264

## Description

**Changed**

- This PR addresses all of the design issues seen on production and ensures that this landing page aligns with the design specs for all breakpoints.
- Add fluent container 
- **Context:** The Merative.com Blog (https://www.merative.com/blog) has been launched for a while now, however, during the development phase of this project, there we no proper design QA review implemented, which resulted in container alignment and spacing issues.

## Design Specs

- Figma Link - https://www.figma.com/file/FSqQ1LYrVSssEFOZuOnMIE/Final-page-layouts-for-Merative.com?type=design&node-id=202%3A67025&mode=dev 

## Test URLs

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/blog
- After (Changes from this PR): https://fix-blog-landing-design-qa--merative2--proeung.hlx.page/blog
  
## Testing Instruction

- Visit this example page - https://fix-blog-landing-design-qa--merative2--proeung.hlx.page/blog
- Ensure that the changes align with the design specs for all of the support breakpoints
